src/sentry/integrations/pagerduty/actions/notification.py

```diff
@@ -54,6 +54,8 @@ def send_notification(event, futures):
                         "error": str(e),
                         "service_name": service.service_name,
                         "service_id": service.id,
+                        "project_id": event.project_id,
+                        "event_id": event.event_id,
```

Comment:
added more logging so that the `fail` and `success` logs have the same kind of information